x-pack/solutions/observability/plugins/apm/public/components/shared/focused_trace_waterfall/index.tsx

```diff
@@ -68,7 +69,16 @@ function getTraceItems(items: NonNullable<FocusedTrace['traceItems']>) {
 export function FocusedTraceWaterfall({ items, onErrorClick }: Props) {
   const reparentedItems = reparentDocumentToRoot(items.traceItems);
   if (!reparentedItems) {
-    return null;
+    return (
+      <EuiCallOut
+        title={i18n.translate('xpack.apm.focusedTraceWaterfall.euiCallOut.incompleteTraceLabel', {
+          defaultMessage:
```

Comment:
We might want to create a section in https://www.elastic.co/docs/troubleshoot/observability/apm/common-problems to help users understand this problem and possibly troubleshoot it. The message alone might not be helpful

Reply:
This makes sense to me. If we decide to go in this direction, can someone (maybe @Bluefinger or @iblancof) open an issue in the [docs content](https://github.com/elastic/docs-content/issues) repo.